src/fu-util.c

```diff
@@ -3793,6 +3794,43 @@ fu_util_show_plugin_warnings(FuUtilPrivate *priv)
 	}
 }
 
+static gboolean
+fu_util_get_bios_attr(FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	g_autoptr(GPtrArray) attrs = NULL;
+	gboolean found = FALSE;
+
+	attrs = fwupd_client_get_bios_attrs(priv->client, priv->cancellable, error);
+	if (attrs == NULL)
+		return FALSE;
+	if (priv->as_json)
+		return fu_util_get_bios_attr_as_json(values, attrs, error);
+
+	for (guint i = 0; i < attrs->len; i++) {
+		FwupdBiosAttr *attr = g_ptr_array_index(attrs, i);
+		if (fu_util_bios_attr_matches_args(attr, values)) {
+			g_autofree gchar *tmp = fu_util_bios_attr_to_string(attr, 0);
+			g_print("%s\n", tmp);
+			found = TRUE;
+		}
+	}
+	if (attrs->len == 0) {
+		g_set_error_literal(error,
+				    FWUPD_ERROR,
+				    FWUPD_ERROR_NOTHING_TO_DO,
+				    _("This system doesn't support firmware settings"));
```

Comment:
Here, and a few more: maybe a `TRANSLATORS:` comment?

Reply:
fixed